apps/pom.xml

```diff
@@ -586,8 +562,39 @@ checkSecretFilesForPlainText() {
 }
 
 checkSecretFilesForPlainText
-checkJavaFormat
 							]]></pre-commit>
+							<pre-push><![CDATA[
+#!/usr/bin/env bash
+set -e
+
+checkJavaFormat() {
+  echo 'Verifying Java source code format...'
+  branch_name="$(git rev-parse --abbrev-ref HEAD 2>&1)"
+
+  git fetch origin 2>&1
+
+  localChanges="$(git diff --name-only "origin/${branch_name}:apps" "${branch_name}:apps" 2>&1)"
+
+  echo "Verifying Java source code format for branch: ${branch_name} ..."
+  if [ "$localChanges" = '' ]; then
```

Comment:
obviously a matter of preference, but curious...
So if 'no changes detected', why not just exit right there? minimizes 'if' nesting resulting in less indentqation

Reply:
we could but didnt want to replicate two return 0's